test-case/manual/DPMST.md

```diff
@@ -0,0 +1,67 @@
+# DPMST test
+DisplayPort Multi-Stream Transport is a manual test.
+
+# Preconditions
+1. Hardware requirements met for each DPMST case
+
+# Test Description
+* Test output from one test unit to two separate monitors with audio output via
+  DP (as well as with USB-C and HDMI adapters if monitors have those inputs
+  available).
+
+# 1) Single DP -> Multi DP hub Test Case:
+## Hardware Requirements
+1. One DisplayPort hub = single DP -> multi-DP
+2. Two monitors with DP input and audio output
+3. Two DP-USB-C adapters (if monitors also have USB-C input)
+4. Two DP-HDMI adapters (if monitors also have HDMI input)
+
+## Run Instructions
+1. Two monitors are connected via one DP hub using DP-DP cables.
+2. Run:
+   ```
+   amixer contents | grep -i jack -A 2
+   ```
```

Comment:
I think we also need some amixer settings for *pin/*port part.  Or we cannot get audio output from each DP monitor.

Reply:
@ClarexZhou indicated that the process no longer includes the *pin/*port info.
> Sof-dev is using common HDMI solution now. Won’t need Pinx-Ports any more.

Reply:
Just confirmed that there're no pin/port kcontrols in amixer setting. now we are using common HDMI, no need to change the amixer settings anymore.